ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -147,42 +181,12 @@ define([
             var definition = this.metacardTypes[id];
             return definition ? definition.alias || id : id;
         },
+        getMetacardStartingTypes: function(){
+            return metacardStartingTypes;
+        },
         metacardDefinitions: [],
         sortedMetacardTypes: [],
-        metacardTypes: {
-            anyText: {
-                id: 'anyText',
-                type: 'STRING',
-                multivalued: false
-            },
-            anyGeo: {
-                id: 'anyGeo',
-                type: 'LOCATION',
-                multivalued: false
-            },
-            'metacard-type': {
-                id: 'metacard-type',
-                type: 'STRING',
-                multivalued: false,
-                readOnly: true
-            },
-            'source-id': {
-                id: 'source-id',
-                type: 'STRING',
-                multivalued: false,
-                readOnly: true
-            },
-            cached: {
-                id: 'cached',
-                type: 'STRING',
-                multivalued: false
-            },
-            'metacard-tags': {
-                id: 'metacard-tags',
-                type: 'STRING',
-                multivalued: true
-            }
-        },
+        metacardTypes: Object.assign({}, metacardStartingTypes),
```

Comment:
❗️ I don't think IE11 supports `Object.assign`

Reply:
It doesn't, but this this works. :trollface: 

```
function mergeObject(target) {
    for (var i = 1; i < arguments.length; i++) {
        var source = arguments[i];
        for (var key in source) {
            if (source.hasOwnProperty(key)) {
                target[key] = source[key];
            }
        }
    }
    return target;
}
```